### PR TITLE
Bump telemeter

### PR DESCRIPTION
**Pull request by squat**
This commit bumps Telemeter client to ensure we are whitelisting the metrics that have recently been added.

cc @brancz @s-urbaniak

### CODE REVIEW

File: assets/telemeter-client/secret.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
+  match-rules: e19fbmFtZV9fPSJ1cCJ9CntfX25hbWVfXz0iY2x1c3Rlcl92ZXJzaW9uIn0Ke19fbmFtZV9fPSJjbHVzdGVyX3ZlcnNpb25fYXZhaWxhYmxlX3VwZGF0ZXMifQp7X19uYW1lX189ImNsdXN0ZXJfb3BlcmF0b3JfdXAifQp7X19uYW1lX189ImNsdXN0ZXJfb3BlcmF0b3JfY29uZGl0aW9ucyJ9CntfX25hbWVfXz0iY2x1c3Rlcl92ZXJzaW9uX3BheWxvYWQifQp7X19uYW1lX189ImNsdXN0ZXJfdmVyc2lvbl9wYXlsb2FkX2Vycm9ycyJ9CntfX25hbWVfXz0ibWFjaGluZV9jcHVfY29yZXMifQp7X19uYW1lX189Im1hY2hpbmVfbWVtb3J5X2J5dGVzIn0Ke19fbmFtZV9fPSJldGNkX29iamVjdF9jb3VudHMifQp7X19uYW1lX189IkFMRVJUUyIsYWxlcnRzdGF0ZT0iZmlyaW5nIn0Ke19fbmFtZV9fPSJjb2RlOmFwaXNlcnZlcl9yZXF1ZXN0X2NvdW50OnJhdGU6c3VtIn0Ke19fbmFtZV9fPSJrdWJlX3BvZF9zdGF0dXNfcmVhZHk6ZXRjZDpzdW0ifQp7X19uYW1lX189Imt1YmVfcG9kX3N0YXR1c19yZWFkeTppbWFnZV9yZWdpc3RyeTpzdW0ifQ==
```
Comment (brancz) [inline assets/telemeter-client/secret.yaml]: just a side note, I think there is a `stringData` field in the yaml that we can make use of as well, maybe that would make diffs like these more review-able

**Comment by squat** [comment]
Travis check has passed but status is not updated here

**Pull request merged**